src/lib/charts/chart.css

```diff
@@ -2,16 +2,26 @@
   &-label {
     display: flex;
     align-items: center;
+    white-space: nowrap;
     @apply lc-text-sm;
     @apply lc-text-black;
-    @apply lc-font-bold;
+
+    &-content {
+      @apply lc-font-bold;
+    }
 
     &-color {
       display: inline-block;
-      width: theme('spacing.8');
-      height: theme('spacing.8');
+      width: 11px;
+      height: 11px;
```

Comment:
@jaredhill4 11px seemed to fit best with the font size and not be too small.

Reply:
Let's go with `12px`. Looks like that's what Marcus intended for the legend for the doughnut graphs. I used `8px` cause that's what was in the tooltip in the mockup.